db/schema.rb

```diff
@@ -15,6 +15,17 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "books", force: :cascade do |t|
+    t.string "title", null: false
+    t.string "author", null: false
+    t.string "image", null: false
+    t.string "editor", null: false
+    t.string "genere", null: false
```

Comment:
you change the field on the model and the migration but at the schema is the same, run migration again to change this

Reply:
done!